OrbitCore/LinuxUtils.cpp

```diff
@@ -129,12 +129,8 @@ void ListModules(pid_t pid,
   }
 
```

Comment:
You will need to parse flags in tokens[1] they have following format "rwxp" where every position can be '-'. the executable flag is in 3rd position.

So the map entry is executable if tokens[1][2] == 'x'

Reply:
Thanks. I added a check and discard them already here. I left the below check, just to be sure.